core/tests/puppeteer-acceptance-tests/specs/practice-question-submitter/submit-practice-questions-with-different-interactions-and-difficulties.spec.ts

```diff
@@ -96,7 +96,9 @@ describe('Question Submitter', function () {
       await questionSubmitter.addImageInteraction();
 
       await questionSubmitter.addHintToState('Test Hint 1');
-      await questionSubmitter.editDefaultResponseFeedback();
+      await questionSubmitter.editDefaultResponseFeedbackInQuestionSubmitter(
```

Comment:
editDefaultResponseFeedbackInQuestionEditorPage is better

Reply:
Done.